domains/homehosting.json

```diff
@@ -0,0 +1,12 @@
+{
+	"description": "Student hosting for SSH devices without open ports.",
+	"owner": {
+		"username": "GenerelSchwerz",
+		"email": "generelschwerz@gmail.com",
+	},
+	"record": {
+		"A": [
+			"156.146.99.77"
```

Comment:
you need to put a website for the domain to be accepted

Reply:
I cannot provide a website as I wish to use the domain provided here as the domain that cloudflare will redirect to. 

[Documentation for what I am attempting to do](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/ssh/#connect-to-ssh-server-with-cloudflared-access)

Basically, I need a domain that can have its nameservers changed to cloudflare's. Once that's done, cloudflare will route everything to an IP that normally cannot have ports open. 

TL:DR; I cannot provide a service until a domain with modifiable nameservers is provided; after that is done, the service will be available. Is that possible with this service?

Reply:
> until a domain with modifiable nameservers is provided

We do not support NS records.